README.md

```diff
@@ -2,231 +2,12 @@
-
-```javascript
-setup({
-  mongoDBUrl: 'mongodb://localhost:27017/my-custom-database',
-}).then(RadiksController => {
-  app.use('/radiks', RadiksController);
-});
-```
-
-### Accessing the MongoDB Collection
-
-#### Using `getDB` to manually connecting to the MongoDB collection
-
-Radiks-server keeps all models inside of a collection. You can use the `getDB` function to access this collection. [See the MongoDB Collection reference](https://mongodb.github.io/node-mongodb-native/3.1/api/Collection.html) for documentation about how you can interact with this collection.
-
-```js
-const { getDB } = require('radiks-server');
-
-const mongo = await getDB(MONGODB_URL);
-```
-
-#### Migration from Firebase (or anywhere else)
-
-Migrating data from Firebase to Radiks-server is simple and painless. You can create a script file to fetch all the firebase data using their API. Then, you can use your MONGOD_URI config to use the `mongodb` npm package.
-
-```js
-// Script for transfering users from Firebase to Radiks-server
-
-const { getDB } = require('radiks-server');
-const { mongoURI } = require('......'); // How you import/require your mongoURI is up to you
-
-const migrate = async () => {
-  // `mongo` is a reference to the MongoDB collection that radiks-server uses.
-  // You can add or edit or update data as necessary.
-  const mongo = await getDB(mongoURI);
-
-  /**
-   * Call code to get your users from firebase
-   * const users = await getUsersFromFirebase();
-   * OR grab the Firebase JSON file and set users to that value
-   * How you saved your user data will proably be different than the example below
-   */
-
-  const users = {
-    '-LV1HAQToANRvhysSClr': {
-      blockstackId: '1N1DzKgizU4rCEaxAU21EgMaHGB5hprcBM',
-      username: 'kkomaz.id',
-    },
-  };
-
-  const usersToInsert = Object.values(users).map(user => {
-    const { username } = user;
-    const doc = {
-      username,
-      _id: username,
-      radiksType: 'BlockstackUser',
-    };
-    const op = {
-      updateOne: {
-        filter: {
-          _id: username,
-        },
-        update: {
-          $setOnInsert: doc,
-        },
-        upsert: true,
-      },
-    };
-    return op;
-  });
-
-  await mongo.bulkWrite(usersToInsert);
-};
-
-migrate()
-  .then(() => {
-    console.log('Done!');
-    process.exit();
-  })
-  .catch(error => {
-    console.error(error);
-    process.exit();
-  });
-```
-
-### Options
-
-You can specify some options while initiating the Radiks server.
+Radiks-server is a pre-built server to index and serve data that lives in decentralized services. Specifically, it is built to index data that is stored in [Gaia](https://github.com/blockstack/gaia), and created using the front-end companion library, [radiks.js](https://github.com/hstove/radiks). Because Gaia is just a key-value store, many applications end up needing to store an 'index' of all underlying data, in order to query it in a performant and flexible way.
```

Comment:
The link should be https://github.com/blockstack/radiks instead of https://github.com/hstove/radiks